### PR TITLE
Hotfix / Character Picker

### PR DESCRIPTION
**Pull request by onlymejosh**
Character Picker now shows all characters for that letter but disabled the buttons on the ones already in use. 

Also fixes an issue with duplication of characters

### CODE REVIEW

File: src/js/steps/letters/generateCharPicker.js [hunk 1]
```diff
@@ -152,7 +155,11 @@ module.exports = function (options, monkeyContainer) {
           // For each of the remaining characters (note, even if there are no
           // other characters to select, this will still render the currently
           // selected character.)
-          $(remainingLetterChars).each(function (ix, charObj) {
+
+        // For each of the the letter characters check to see if its already
+        // being used. We show all of the characters at all points so its clear
+        // which ones are available.
```
Comment (callumacrae) [inline src/js/steps/letters/generateCharPicker.js]: Indentation is off, are you using tabs?

Comment (callumacrae) [inline src/js/steps/letters/generateCharPicker.js]: Indentation is off, are you using tabs?

File: src/js/steps/letters/init.js [hunk 2]
```diff
@@ -371,35 +372,35 @@ module.exports = function ($events, options, $monkeyContainer) {
...
-      $buttonEl.find('.button')
-        .attr('disabled', true)
-        .removeClass('primary')
-        .text('In Use');
-      $buttonEl.addClass('selected-char');
+      $disableButtons.forEach(function($button) {
+        $button.find('.button')
+          .attr('disabled', true)
+          .removeClass('primary')
+          .text('In Use');
```
Comment (callumacrae) [inline src/js/steps/letters/init.js]: Inline language should be passed in in the lang options

**Comment by tomcartwrightuk** [comment]
@onlymejosh Closing this PR as we have the other one open with same functionality

**Pull request closed without merging**